cs3api4lab/tests/share_test_base.py

```diff
@@ -188,3 +190,37 @@ def remove_test_file(self, user, file_path):
 
     def get_random_suffix(self):
         return '-' + ''.join(random.choices(string.ascii_lowercase + string.digits, k=5))
+
+    def clean_up_file(self, user, file_name):
+        try:
+            self.remove_test_file(user, file_name)
+        except Exception as ex:
+            print(str(ex))
+
+    def clean_up_share(self, user, share_id):
+        try:
+            self.remove_test_share(user, share_id)
+        except Exception as ex:
+            print(str(ex))
+
+    def remove_share_and_file_by_path(self, user, file_path):
+        if user == 'einstein':
+            share_api = self.share_api
+            storage = self.storage_logic
+        if user == 'marie':
+            share_api = self.marie_share_api
+            storage = self.marie_storage_logic
+        if user == 'richard':
+            share_api = self.richard_share_api
+            storage = self.richard_storage_logic
+
+        stat = storage.stat(file_path, None)
+        if stat is None:
```

Comment:
This was because you could not remove a share if the file didn't exist? If so, is this documented? Maybe here would be helpful as well.

Reply:
ok I've addeed a comment